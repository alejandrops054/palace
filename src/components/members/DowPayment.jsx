/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react';
import { Select } from "../index";

const DowPayment = () =>{
    return(
        <>
        <div className="container">
            <div className="card">
                <div className="row">
                    <div className="col-md-6">
                        <h3>Dow Payment Other</h3>
                    </div>
                    <div className="col-md-6">
                        <h3>Cash out</h3>
                    </div>
                </div>
                   <div className="row">
                        <div className="col-md-6">
                            <Select
                                type="text"
                                label="City paid"
                                placeholder="$0.00"
                                col={12}
                            />
                        </div>
                        <div className="col-md-6">
                            <Select
                                type="text"
                                label="Cash options"
                                placeholder="$0.00"
                                col={12}
                            />
                        </div>
                   </div>
                </div>
            </div>
        </>
    )
}

export default DowPayment;