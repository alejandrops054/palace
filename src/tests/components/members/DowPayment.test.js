/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import DowPayment from '../../../components/members/DowPayment'

describe('DowPayment component',() => {
    test('debe mostrar snapshot', () => {
        
        const wrapper = shallow(<DowPayment/>)
        expect(wrapper).toMatchSnapshot();
    })
    
})