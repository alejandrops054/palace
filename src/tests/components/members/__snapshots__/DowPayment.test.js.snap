// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DowPayment component debe mostrar snapshot 1`] = `
<Fragment>
  <div
    className="container"
  >
    <div
      className="card"
    >
      <div
        className="row"
      >
        <div
          className="col-md-6"
        >
          <h3>
            Dow Payment Other
          </h3>
        </div>
        <div
          className="col-md-6"
        >
          <h3>
            Cash out
          </h3>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-md-6"
        >
          <Select
            col={12}
            label="City paid"
            placeholder="$0.00"
            type="text"
          />
        </div>
        <div
          className="col-md-6"
        >
          <Select
            col={12}
            label="Cash options"
            placeholder="$0.00"
            type="text"
          />
        </div>
      </div>
    </div>
  </div>
</Fragment>
`;
